api/tools/src/main/java/org/pmiops/workbench/tools/FetchWorkspaceDetails.java

```diff
@@ -26,7 +30,14 @@
  * workspaces found.
  */
 @Configuration
-@Import({FireCloudServiceImpl.class, FireCloudConfig.class})
+@Import({
```

Comment:
all of these imports are necessary.  unclear why that would have been different in the past.